content/en/docs/v3.5/op-guide/configuration.md

```diff
@@ -248,11 +248,9 @@ The security flags help to [build a secure etcd cluster][security].
 
 ### --ca-file
 
-**DEPRECATED**
+**REMOVED**
 
-+ Path to the client server TLS CA file. `--ca-file ca.crt` could be replaced by `--trusted-ca-file ca.crt --client-cert-auth` and etcd will perform the same.
-+ default: ""
-+ env variable: ETCD_CA_FILE
+This flag has been removed. `--trusted-ca-file ca.crt --client-cert-auth` can be used to perform the same task as was performed by `--ca-file ca.crt` previously. 
```

Comment:
```suggestion
+ This flag has been removed. `--trusted-ca-file ca.crt --client-cert-auth` can be used to perform the same task as was performed by `--ca-file ca.crt` previously.
```
Let's add a bullet point for consistency (and remove the extra space at the end of the line).